src/lib/routes/admin-api/config.ts

```diff
@@ -153,7 +139,8 @@ class ConfigController extends Controller {
         const flags = {
             ...this.config.ui.flags,
             ...expFlags,
-            displayUpgradeEdgeBanner: usesOldEdge,
+            displayUpgradeEdgeBanner:
+                this.flagResolver.isEnabled('edgeBulkMetrics'),
```

Comment:
Should this just be:
```suggestion
                this.flagResolver.isEnabled('upgradeEdgeBanner'),
```

And can't we just use the existing banner functionality?

Reply:
I was thinking the same thing as Gaston. We should leave this as is (mostly for OSS users).

Reply:
Hmm, for OSS we've already disabled the endpoint, so old edge won't have given you metrics, but it should give the banner. But after the last couple of days with testing the usesSdkOlderThan doesn't seem to work as well in practice as it did in our testing.

With regards to using existing banner functionality, the assignment back in 5.9 was to write a specific banner component so we could show it to OSS users/self-hosted as well.

Reply:
> But after the last couple of days with testing the usesSdkOlderThan doesn't seem to work as well in practice as it did in our testing.

I experienced the banner in enterprise while playing around with edge about the token creation. It does have an odd behavior but it shows up. Maybe we can look at this together.

> With regards to using existing banner functionality, the assignment back in 5.9 was to write a specific banner component so we could show it to OSS users/self-hosted as well.

I assume now we are changing to showing this to only hosted instances (not OSS nor self-hosted). At least, that's what I understand from this PR. But maybe it's 2 fold: we keep the existing functionality for OSS and self-hosted and we have a toggle for hosted, but if we're doing that for hosted, we can use the existing banner functionality with the flag: messageBanner

Reply:
Yeah, reverted the change that removed the usage of checking for client instances. Now the PR literally is just adding the possibility to toggle the banner on explicitly through the `displayEdgeBanner` flag